modules/auxiliary/analyze/crack_databases.rb

```diff
@@ -208,15 +208,9 @@ def check_results(passwords, results, hash_type, hashes, method, cracker)
     # Inner array format: db_id, hash_type, username, password, method_of_crack
     results = []
 
-    cracker = new_password_cracker
+    cracker = new_password_cracker(action.name)
     cracker.cracker = action.name
```

Comment:
based on all the other changes, i think you missed removing this line

Reply:
Good catch! Fixed.